src/main/java/cart/controller/AdminController.java

```diff
@@ -0,0 +1,37 @@
+package cart.controller;
+
+import cart.dto.ProductRequestDto;
+import cart.service.AdminService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/admin")
+public class AdminController {
+
+    private final AdminService adminService;
+
+    public AdminController(AdminService adminService) {
+        this.adminService = adminService;
+    }
+
+    @PostMapping("/products")
+    public ResponseEntity<String> addProduct(@Valid @RequestBody ProductRequestDto productRequestDto) {
```

Comment:
```suggestion
    public ResponseEntity<Void> addProduct(@Valid @RequestBody ProductRequestDto productRequestDto) {
```

단순히 ok만 응답하기 때문에 리턴타입을 Void로 해도 괜찮을 것 같아요